bundles/org.openhab.binding.homewizard/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -98,6 +131,13 @@
 
 	</thing-type>
 
+
+	<channel-type id="power_failures">
+		<item-type>Number</item-type>
+		<label>Any Power Failures</label>
```

Comment:
For consistency, I think this should now be:
```suggestion
		<label>Power Failures</label>
```

?

Reply:
Yep for sure!